docs/en/query_language/syntax.md

```diff
@@ -98,35 +108,35 @@ For more information, see the section "Operators" below.
 
 Data types and table engines in the `CREATE` query are written the same way as identifiers or functions. In other words, they may or may not contain an arguments list in brackets. For more information, see the sections "Data types," "Table engines," and "CREATE".
 
-## Expression Aliases
+## Expression Aliases {#syntax-expression_aliases}
 
-Alias is a user defined name for an expression in a query.
+Alias is a user defined identifier for an expression in a query.
```

Comment:
actually "name" was more easy to understand, nobody cares that technically it's identifier (which could be clarified later)